### ISSUE TITLE
Great job on this integration! Question about Climate control & Sentry mode

### ISSUE THREAD

**Issue by psmith3** [UniversalDevicesInc/tesla-nodeserver#1]
Thanks so much for creating this integration. Have to working on my ISY994i now. Do you plan to integrate the ability to start and stop the climate & set temperature? Also would like to be able to control activating Sentry mode. Will be glad to beta test anything.

**Comment by mkohanim** [comment]
HI Parker,

Thanks so very much for the feedback. At the moment, we are just checking to see what type of feedback we get and how many would actually use this node server. Once we have a better understanding, we’ll definitely consider adding/enhancing features.

With kind regards,

******************************
  Michel Kohanim
  CEO

  (p) 818.631.0333
  http://www.universal-devices.com<http://www.universal-devices.com/>
******************************

**Comment by psmith3** [comment]
Thanks. I would specifically like to have my climate control automatically turn on when I press a keypad button inside the house, etc. Also, there is a need to manually activate sentry mode today and it would be nice to automatically enable at a certain time of day.

**Comment by mkohanim** [comment]
Hi Parker,

Wouldn’t all those cause Tesla not to go to sleep?

With kind regards,

******************************
  Michel Kohanim
  CEO

  (p) 818.631.0333
  http://www.universal-devices.com<http://www.universal-devices.com/>
******************************

**Comment by psmith3** [comment]
Correct. Fine with me when I am at home. Usually plugged up to charger at home, overnight.

**Comment by mkohanim** [comment]
Hi Parker, thank you. We’ll definitely consider as soon as we have some more data.

With kind regards,

******************************
  Michel Kohanim
  CEO

  (p) 818.631.0333
  http://www.universal-devices.com<http://www.universal-devices.com/>
******************************

**Comment by blakevh** [comment]
I second this request for turning on the interior conditioning.   It was the first thing I thought of when I found out that the Tesla API existed.   Even wrote some prototype calls from my ISY to initiate it, but the wakeup always was a problem.
One of the major features for cold weather Tesla owners is to pre-condition the vehicle while it is on line power.  It is actually a recommendation from Tesla to warm up your car about an hour before driving for multiple reasons.
1. It gets the battery up to temperature on line power.
2. It gets the car interior warmed up on line power - and warming up the interior is important to unfreeze the door handles and windows.

And there is a timeout on the conditioning (heating or cooling) so it does not stay on indefinitely.

I also use Alexa, so wouldn't it just be so cool to say, Alexa, warm up the Tesla!  I know, we will probably have to say Alexa, turn on Tesla or something similar, but you get my idea.

By the way, if you are interested, I can create a pull request as I played with the code to add the CLIMATE_ON, CLIMATE_OFF,  and the interior temperature status.
[tesla-nodeserver.zip](https://github.com/UniversalDevicesInc/tesla-nodeserver/files/3169556/tesla-nodeserver.zip)
